### ISSUE TITLE
[Bug]: In Applicant Tracker template all the widgets in container1 fails to get highlighted 

### ISSUE THREAD

**Issue by Richarex** [appsmithorg/appsmith#16177]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When the user forks the 'Applicant Tracker' template on the workspace it is observed that all the widgets in 'Container1' fails to get highlighted for the user.

### Steps To Reproduce

1. Go to url:- https://release.app.appsmith.com/
2. Click on 'Templates' and on 'Applicant Tracker' template click on Fork icon 
3. Now on 'Choose where to fork the template' dialog page select your workspace & click on "Fork Template".
4. On your workspace observe the 'Container1' widgets

![chrome_vf7quKIMEf](https://user-images.githubusercontent.com/93259714/185849490-9a4cc541-8ba0-4fd0-8aba-f932447064aa.png)
https://user-images.githubusercontent.com/93259714/185849313-9ca9e95b-9555-451a-bcd4-b47ec22f87ad.mp4

### Version

Cloud - 1.7.14

**Comment by kocharrahul7** [comment]
Hey @Richarex can you try to select a user from the table? The container widgets are supposed to be hidden by default. If that is broken, we can work for a fix

**Comment by Richarex** [comment]
@kocharrahul7 Yeah it worked after selecting any one user. But still one question how will the user know that they are suppose to select the user from the table can we have a tool-tip to help?
